### PR TITLE
Updating repo to adhere to Vonage repo-standards

### PR DESCRIPTION
**Pull request by MichaelJolley**
Includes:

- Updated README.md to include branding change image
- Updated README.md to define additional support options
- Added CODE_OF_CONDUCT.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # OpenTok Java SDK
 
 [![Build Status](https://travis-ci.org/opentok/Opentok-Java-SDK.svg?branch=master)](https://travis-ci.org/opentok/Opentok-Java-SDK)
-[![codecov](https://codecov.io/gh/opentok/Opentok-Java-SDK/branch/master/graph/badge.svg)](https://codecov.io/gh/opentok/Opentok-Java-SDK)
+[![codecov](https://codecov.io/gh/opentok/Opentok-Java-SDK/branch/master/graph/badge.svg)](https://codecov.io/gh/opentok/Opentok-Java-SDK) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
```
Comment (slorello89) [inline README.md:4]: let's maybe drop the codecov badge while we don't know what the coverage is.

File: README.md [hunk 2]
```diff
@@ -199,10 +201,10 @@ The `Archive.OutputMode.COMPOSED` option is the default value for `outputMode`.
 You can only specify the `resolution` for composed archives using the `ArchiveProperties` builder. If you set the `resolution` property and also set the `outputMode` property to `Archive.OutputMode.INDIVIDUAL`, the method will throw an `InvalidArgumentException`.
 
 The accepted values for `resolution` are:
+
```
Comment (slorello89) [inline README.md:204]: was this extra white-space intended?

Comment (MichaelJolley) [reply README.md:204]: It wasn't done intentionally.  Was likely my markdown linter.

File: README.md (hunk 1 above)
Comment (MichaelJolley) [reply README.md]: Done

**Pull request merged**